src/Control.cpp

```diff
@@ -733,7 +744,7 @@ void Control::switchTo(Object* theTarget) {
           // "Seclusion: Islesbury" uses a custom behaviour for leaving Slides.
           // As a result this change was an easy fix to the problem of the game
           // being unable to use direct control mode after leaving a Slide.
-          // _directControlActive = false;
+          _directControlActive = false;
```

Comment:
Note that this undoes the change we did for how slides are handled in Seclusion. Obviously this breaks Seclusion, but if we don't undo it, everything else is broken. In other words, we have to come up with a proper solution to Seclusion's problem so we can get rid of this hack.